pkgs/applications/gis/udig/default.nix

```
@@ -0,0 +1,66 @@
+{ stdenv, fetchurl, unzip, makeWrapper, jre8, libXtst, gdal }:
+let
+  pname = "udig";
+  version = "2.0.0";
+
+  srcs = {
+    x86_64-linux = fetchurl {
+      url = "http://udig.refractions.net/files/downloads/udig-${version}.linux.gtk.x86_64.zip";
+      sha256 = "03hj1mdd6sq0gbpa838wkccibp3l2hvnwxxf5dyc0jk3mmd94fwa";
+    };
+    x86_64-darwin = fetchurl {
+      url = "http://udig.refractions.net/files/downloads/udig-${version}.macosx.cocoa.x86_64.zip";
+      sha256 = "16rcyp1zy3lr1hwjhzh6vwcgck52w66dm1qsc52gppy1f4i3f692";
+    };
+  };
+  src = srcs.${stdenv.hostPlatform.system};
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```

per recent contributing guide change.